### PR TITLE
Added react sample

### PR DESCRIPTION
**Pull request by Metastasis**
### Description

Resolves "React sample" from #6 

@Siemienik 
Do we need to run cypress tests in Github Actions?

### CODE REVIEW

**Comment by Siemienik** [comment]
> Do we need to run cypress tests in Github Actions?

It will be great to do this, I may write a workflow if want you 👍

**Comment by Siemienik** [comment]
At first look, code looks fine. I'm going to review it again tomorrow - and run this app locally :) Thank you :)

File: samples/samples-react/public/index.html
Comment (Metastasis) [inline samples/samples-react/public/index.html:32]: Add this as dependencies and import them from index,css

**Comment by Metastasis** [comment]
@Siemienik 
So GHA testing over samples will be done in separate issue?

**Comment by Siemienik** [comment]
> @Siemienik
> So GHA testing over samples will be done in separate issue?

yes

File: samples/samples-react/package.json
Comment (Siemienik) [inline samples/samples-react/package.json:5]: As you are an author, please add your info here 😄

File: samples/samples-react/README.md
Comment (Siemienik) [inline samples/samples-react/README.md:18]: Please add information about useful scripts, like run cypress test runner, start the app, etc

File: README.md
Comment (Siemienik) [inline README.md:109]: please move to `./samples/react/`

File: samples/samples-react/public/index.html
Comment (Siemienik) [inline samples/samples-react/public/index.html:3]: samples/samples-react/public/logo192.png
and samples/samples-react/public/logo512.png

are not need anymore, could I ask you to delete?

File: samples/samples-react/src/App.js
Comment (Siemienik) [inline samples/samples-react/src/App.js:51]: please add info that is reactjs sample with link into REPO/samples/react

File: samples/samples-react/src/App.js
Comment (Siemienik) [inline samples/samples-react/src/App.js:49]: could I propose to add GH icon with link into repo?

File: samples/samples-react/src/components/Invoice/Invoice.jsx
Comment (Siemienik) [inline samples/samples-react/src/components/Invoice/Invoice.jsx:1]: It looks pretty good 🥰

**Comment by Siemienik** [comment]
hmm.. I have idea to publish it like gh-page. WDYT?

File: samples/samples-react/src/App.js
Comment (Metastasis) [reply samples/samples-react/src/App.js:49]: Where icon should be added?

**Comment by Metastasis** [comment]
> > Do we need to run cypress tests in Github Actions?
> 
> It will be great to do this 👍 , I may write a workflow if want you ? (I'm going to create an issue to increase GHA testing over samples)
@Siemienik 
Yes, it would be great. I think I have to change npm test scripts in order to work with GHA
I think it should look something like this
What do you think?
```json
{
  "scripts": {
    "cy:run:chrome": "cypress run --browser chrome --headless",
    "cy:run:firefox": "cypress run --browser firefox --headless",
    "test:chrome": "start-server-and-test start-server http://localhost:3000 cy:run:chrome",
    "test:firefox": "start-server-and-test start-server http://localhost:3000 cy:run:firefox",
    "test:jest": "react-scripts test --watchAll=false --runInBand",
    "test": "npm-run-all test:* --parallel --bail",
  }
}
```

File: README.md
Comment (Metastasis) [reply README.md:109]: yeah, I'll fix it, it's just CRA, it wont allow you to create project with this name

**Comment by Metastasis** [comment]
> hmm.. I have idea to publish it like gh-page. WDYT?

Good idea 👍

File: samples/samples-react/src/App.js
Comment (Siemienik) [reply samples/samples-react/src/App.js:49]: I hadn't specified place. What are you thing about fixed right top?

**Comment by Siemienik** [comment]
> PS: just realized that two servers will be started on the same port, I need to think how I can workaround this

Usually I realised it by:

```yml
strategy:
  matrix:
    test-cmd: [jest, firefox, chrome]

# ... in a step
run: npm run test:${{ strategy.matrix.test-cmd }}
```

or (if running server explicitly by `serve`)
```yml
# Unit test step
run: |
  npm run test:unit

# Browser tests
run: |
  npm run serve
  npm run test:browser
```

File: samples/samples-react/src/App.js
Comment (Metastasis) [reply samples/samples-react/src/App.js:49]: Ah, Ok, I'll think of something

**Comment by Metastasis** [comment]
@Siemienik 
I've fixed `npm run test` script so you could run all tests at once in CI environment
You could also run only cypress tests with `npm run cy:run`

**Comment by Siemienik** [comment]
> Do we need to run cypress tests in Github Actions?

I'm start to describe it in #65

**Comment by Metastasis** [comment]
Rebased

**Review by Siemienik — APPROVED** [review]
Pretty great job🥇  all LGTM 👍 

Just for ask, has you possibility to click merge when add approve? If yes, could I please you to merge it by using "Squash and Merge"?
If it is blocked by GH, please bring me back, I attempt to unlock this for PRs with approves.

**Comment by Metastasis** [comment]
@Siemienik 
I can't see such button

**Comment by Siemienik** [comment]
Thank you for information. I really love squash merging however it rewrite commit autor to someone who press down the button. Thank you for rebasing branch before merge ;)

**Pull request merged**
